src/index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+	<meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
+	<meta http-equiv="Pragma" content="no-cache" />
+	<meta http-equiv="Expires" content="0" />
+	<script src="bundle.js" type="text/javascript"></script>
+</head>
+
+<body>
+	<div id="container">
+
+	<!-- Start your code from here -->
+	<header>
+        <div>
+            <img src="https://www.tailorbrands.com/wp-content/uploads/2020/07/twitter-logo.jpg" alt="" class="logo">
+        </div>
+        <form action="">
+            <input type="text" name="search" class="search-here" placeholder="You can search here">
```

Comment:
I am not able to see the word document structure after executing your code you have created every element as block level element please create the structure according to word document